src/main/java/nl/tudelft/lifetiles/graph/models/jgrapht/JGraphTGraphAdapter.java

```diff
@@ -223,4 +230,21 @@ private void removeEdge(final Edge<V> edge) {
         internalGraph.removeEdge(unpackEdge(edge));
     }
 
+    /**
+     * @author Rutger van den Berg
+     *         Compares two edges by their target vertex.
+     */
+    class EdgeComparatorByVertex implements Comparator<Edge<V>> {
```

Comment:
It is more than one line, and thus too complicated for a lambda(in my opinion). And I may want to use it again. Anonymous class doesn't really add much.